docs/getting-started/cv-deploy-porting.md

```diff
@@ -147,5 +147,56 @@ We recommend leveraging the `cv_server` and `cv_token` keys to specify the authe
 
 ```
 
+## Uploading Static Configlets
+
+The `cv_deploy` role also replaces the functionality of the legacy `cvp_configlet_upload` role. Instead of discovering configlets in a directory, `cv_deploy` uses an explicit manifest to manage configlets and containers in the CloudVision **Static Configuration Studio**.
+
+To manage only the Static Configuration Studio content, you run in **"manifest-only" mode** by setting `cv_devices: []`. This instructs the role to skip all device-specific operations and only process the manifest. Note that the playbook task still requires the standard connection variables and can use other keys like `cv_run_change_control` specified in the previous section.
+
+<div class="grid" markdown>
+
+=== "Old (`cvp_configlet_upload`)"
+
+    The legacy `cvp_configlet_upload` role would scan a specified directory and upload any files with a given extension.
+
+    ```yaml
+    ---
+    - name: Upload CVP configlets
+      ansible.builtin.import_role:
+        name: arista.avd.cvp_configlet_upload
+      vars:
+        configlet_directory: "configlets/"
+        file_extension: "txt"
+        configlets_cvp_prefix: "DC1-AVD_"
+    ```
+
+=== "New (`cv_deploy`)"
+
+    With `cv_deploy`, you define a manifest that explicitly lists each configlet name and source file.
+
+    ```yaml hl_lines="11 14-19"
+    ---
+    - name: Upload CVP configlets to Static Configuration Studio
+      ansible.builtin.import_role:
+        name: arista.avd.cv_deploy
+      vars:
+        cv_server: <hostname or IP address of CloudVision host>
+        cv_token: <insert service_account token here - use Ansible Vault>
+        cv_run_change_control: true
+
+        # Use "manifest-only" mode to skip device-specific tasks.
+        cv_devices: []
+
+        # Explicitly define configlets and containers in a manifest.
+        cv_static_config_manifest:
+          configlets:
+            - name: "DC1-AVD_access_lists"
+              file: "configlets/access_lists.txt"
+            - name: "DC1-AVD_ntp_servers"
+              file: "configlets/ntp_servers.txt"
+    ```
+
+</div>
+
 !!! note
-    For a complete overview of all the updates in the `cv_deploy` role, please see the role [documentation](../../ansible_collections/arista/avd/roles/cv_deploy/README.md#steps-to-create-service-accounts-on-cloudvision).
+    For a complete overview of all the updates and capabilities in the `cv_deploy` role, including how to manage containers, please see the role [documentation](../../ansible_collections/arista/avd/roles/cv_deploy/README.md)
```

Comment:
I would add a link to the specific section for containers in the container section